Espanol/Intro_es.ipynb

```diff
@@ -574,8 +574,8 @@
     "pl.plot(dat['z'], dat['Effective magnitude'], marker='.', lw=0.0)\n",
     "\n",
     "pl.plot(zs, effmag(zs, 0.0, 6.), c='k',   label='No Dark Energy', alpha=0.5)\n",
-    "pl.plot(zs, effmag(zs, 0.7, 6.), c='k',   label='50% Dark Energy!')\n",
-    "pl.plot(zs, effmag(zs, 0.751, 6.), c='c', label='75% Dark Energy!')\n",
+    "pl.plot(zs, effmag(zs, 0.5, 6.), c='k',   label='50% Dark Energy!')\n",
```

Comment:
Was this also wrong in the English?

Reply:
Yes, but no I do not remember if I fixed this on english version :(